code/modules/crafting/recipes.dm

```diff
@@ -344,10 +344,10 @@
 	name = "Chemical Bomb Core (gibtonite)"
 	result = /obj/item/weapon/bombcore/chemical
 	reqs = list(
-		/obj/item/weapon/stock_parts/matter_bin/super = 1,
+		/obj/item/weapon/stock_parts/matter_bin = 1,
 		/obj/item/weapon/twohanded/required/gibtonite = 1,
 		/obj/item/weapon/grenade/chem_grenade = 2
 	)
-	parts = list(/obj/item/weapon/grenade/chem_grenade = 2)
+	parts = list(/obj/item/weapon/stock_parts/matter_bin = 1, /obj/item/weapon/grenade/chem_grenade = 2)
```

Comment:
the same matter bin transfers to it, but payloads aren't currently deconstructable so i just get rid of that information after using it.